apps/src/componentLibrary/toggle/Toggle.tsx

```diff
@@ -8,6 +8,8 @@ import Typography from '@cdo/apps/componentLibrary/typography';
 import moduleStyles from './toggle.module.scss';
 
 export interface ToggleProps {
+  /** Toggle id selector, used for UI tests */
+  id?: string;
```

Comment:
@levadadenys Is this ok with you or is there a better way to assign an `id` to component library components?